src/cli/onefuzz/templates/libfuzzer.py

```diff
@@ -233,3 +234,113 @@ def basic(
         self.logger.info("done creating tasks")
         helper.wait()
         return helper.job
+
+    def merge(
```

Comment:
Will add an option to specify the output directory which could be the initial input directory